### ISSUE TITLE
CollectionView scrolling inside the new tabview is hard

### ISSUE THREAD

**Issue by AswinPG** [xamarin/XamarinCommunityToolkit#591]
### Description
When a collectionview is scrolled inside the tabview with swipe enabled, collectionview does not scroll but the tab gets changed. If there is a slight slant to the users swipe, the tabview carouselview is scrolled instead of the collectionview. In short scrolling inside swipe enabled tabview is hard
### Steps to Reproduce

1. Create a swipe enabled Tabview
2. Add a collectionview inside it
3. Try to scroll

### Expected Behavior
Swiping slightly in a a slight angle must scroll the collectionview and not the carouselview
### Actual Behavior
Swiping slightly in a a slight angle scrolls the carouselview and not the collectionview 
### Basic Information

- Version with issue:  1.0.0-pre5
- Last known good version: None
- IDE: VS2019
- Platform Target Frameworks: Checked android only
  - iOS:  Not checked
  - Android: 10
  - UWP:  Not Checked

### Workaround
None

### Reproduction imagery

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/29278621/99681059-c189c700-2aa3-11eb-9db0-1d5ce9bfc155.gif)

**Comment by AswinPG** [comment]
@jsuarezruiz Can you please share your thoughts on this issue ..?

**Comment by jsuarezruiz** [comment]
You have a TabView with CollectionView on each tab and if you scroll vertically, does it take control of the TabView swipe instead of CollectionView scroll?.  If this happens, it should like a bug, something we must improve.

**Comment by AswinPG** [comment]
Yes exactly. If we scroll up in a straight line then it's ok but a user won't scroll like that, casual scrolling leads to this issue

**Comment by AswinPG** [comment]
> You have a TabView with CollectionView on each tab and if you scroll vertically, does it take control of the TabView swipe instead of CollectionView scroll?. If this happens, it should like a bug, something we must improve.

@jsuarezruiz If you could suggest a way to clear this issue, I'd happy to test and implement it and then contribute to XCT

**Comment by vecalion** [comment]
We're also experiencing this usability issue on Android. Users have to be very precise to be able to scroll vertically the content of the selected tab.

**Comment by AswinPG** [comment]
Has anyone found any workarounds to this ..?

**Comment by Dreamescaper** [comment]
Interestingly enough, it behaves more or less fine with ScrollView. 
But with CollectionView it's mostly unusable.

**Comment by HeikkiDev** [comment]
Same issue using ListView and TabView on Android. 

Only workaround I found is setting 
`IsSwipeEnabled="False"`

Notice this way your force users to tap tab headers to switch to another tab content.

**Comment by Enielezi** [comment]
Is there any update on this. Would there be a way to make Swipe on Tabview less horizontally sensitive?
